community/mei-by-laws.md

```diff
@@ -73,41 +77,51 @@ The Board is self-organizing; that is, there will be no formal regulations on ho
 
 ### 5.1 Members
 
-The Board consists of nine (9) elected members plus one (1) member designated by each Host Institution (for details see 7 Host Institutions). All members of the Board (elected and designated) have the right to participate in decision-making processes established by the Board. Any member of the Board may resign from the Board on written notification to the administrative chair (see below).
-
-### 5.2 Meetings
+The Board consists of nine (9) elected members plus one (1) member designated by each Host Institution (for details see 7 Host Institutions). All members of the Board (elected and designated) have the right to participate in decision-making processes established by the Board.
 
-The Board holds regular, virtual or in-person meetings, e.g. approximately every three months. As schedules and funding permit, the Board meets in person at least once per year. For every meeting, an agenda shall be published online prior to the meeting and minutes of the meeting shall be provided in a timely manner following the meeting. Depending on the availability of funding, the members of the Board may be refunded travel costs for this meeting.
+#### 5.1.1 Appointed offices
 
 The Board assigns the following roles to individuals amongst its members:
 
 * one administrative chair
 * two Technical Co-chairs
+* two Elections Officers
 
-The administrative chair takes responsibility for calling Board meetings. The designated  Host Institution Board members act as _liaison_ with the host organizations. Moreover, the chair of the Board is the main representative of the whole MEI Community and thus has representational functions and obligations and may act on behalf of the whole MEI community on such occasions. (N.B. the above is not to say that the chair of the Board has more influence and decision competence than any other member of the Board. The chair of the Board is simply be “first among equals.”)
+The administrative chair takes responsibility for calling Board meetings. The designated Host Institution Board members act as _liaison_ with the host organizations. Moreover, the chair of the Board is the main representative of the whole MEI Community and thus has representational functions and obligations and may act on behalf of the whole MEI community on such occasions. (N.B. the above is not to say that the chair of the Board has more influence and decision competence than any other member of the Board. The chair of the Board is simply be “first among equals.”)
 
 The two Technical Co-chairs take responsibility for organizing and chairing the Technical Team, see details below (6 Technical Team).
 
-### 5.3 Terms & Elections
+The two Elections Officers will take care of running the yearly MEI elections. The Elections Officers are appointed by the Board from amongst the MEI community and should not be running for election themselves.
```

Comment:
Suggest changing "should" to "shall". 
Better yet, suggest changing "should not be running for election themselves" to "shall not be candidates in the election themselves".
"Should" is a suggestion word. "Shall" is a command word. It seems like a good idea to say it is forbidden, not just suggested, for the Elections Officers to be candidates in the election they run. 
I will note other places in these By-laws where it might be better to replace the word "should" with something stricter.

Reply:
Thank you. Changed to "shall".